byteman/4.0/election_counter_leader_favor_node2.btm

```diff
@@ -7,8 +7,8 @@ RULE election counter leader cheat
 CLASS org.apache.cassandra.service.StorageProxy
 METHOD findSuitableEndpoint
 AT EXIT
-BIND isthere:boolean = $localEndpoints.contains(org.apache.cassandra.locator.InetAddressAndPort.getByName("127.0.0.2"));
+BIND isthere:boolean = $localReplicas.stream().anyMatch(r -> { try { return r.endpoint().equals(org.apache.cassandra.locator.InetAddressAndPort.getByName("127.0.0.2")); } catch (Throwable t) { return false; } });
```

Comment:
localEndpoints are call localReplicas and it is a collection of `Replica`s, not `InetAddress(WithPort)`